### PR TITLE
Added a blueprint util-addon to fix addon generation with pod style

### PR DESCRIPTION
**Pull request by morhook**
This should fix a problem I've reported on https://github.com/ember-cli/ember-cli/issues/5942

### CODE REVIEW

**Comment by morhook** [comment]
Should this modification also go to https://github.com/ember-cli/ember-cli-legacy-blueprints ? I'm a little bit confused by that one

**Comment by rwjblue** [comment]
Looks good to me.

@Turbo87 / @trabus - Would one of y'all mind reviewing also?

**Comment by Turbo87** [comment]
I'm wondering if this should be fixed in https://github.com/emberjs/ember.js/blob/master/blueprints/-addon-import.js instead. Is `util` the only blueprint that is having issues with addon+pod?

**Comment by morhook** [comment]
I've tried to fix https://github.com/emberjs/ember.js/blob/master/blueprints/-addon-import.js with no-luck

There are other blueprints failing pretty similar. After some testing I've found out:
* `ember generate helper help-someone --pod`
* `ember generate initializer start-me-up --pod`

Looks like this cases should be ignored on `-add-on-import.js`? Or maybe we can patch the three of them with helper-addon ?

**Comment by morhook** [comment]
Found another generator that is failing on addon + pod. `ember generate instance-initializer start-me-down --pod`

File: node-tests/blueprints/helper-addon-test.js [hunk 1]
```diff
@@ -20,4 +20,14 @@ describe('Acceptance: ember generate and destroy helper-addon', function() {
...
+  
+  it('in-addon helper-addon foo-bar --pod', function() {
+    var args = ['helper-addon', 'foo-bar', '--pod'];
+
+    return emberNew({ target: 'addon' })
+      .then(() => emberGenerateDestroy(args, _file => {
+        expect(_file('app/helpers/foo-bar.js'))
+          .to.contain("export { default, fooBar } from 'my-addon/helpers/foo-bar';");
+      }));
+  });
```
Comment (Turbo87) [inline node-tests/blueprints/helper-addon-test.js]: it's certainly useful to have these tests here. I'm just not sure what the expected path layout for Pods in these cases is.

/cc @rwjblue @dgeb

**Comment by Turbo87** [comment]
@morhook I might have put you on the wrong path. I actually meant https://github.com/ember-cli/ember-cli/blob/master/blueprints/addon-import/index.js in my first comment 🙈

**Comment by morhook** [comment]
@Turbo87 I think that file should be patched the same way as on this PR to https://github.com/emberjs/ember.js/blob/master/blueprints/-addon-import.js . Seems pretty similar code :laughing:

**Comment by locks** [comment]
Hi @morhook, your contribution is appreciated but as mentioned in https://github.com/ember-cli/ember-cli/issues/5942#issuecomment-242951458, we are not supporting pods, concentrating efforts on the module unification rfc instead. Thanks for the work and hope to see you around.

**Pull request closed without merging**